packages/components/src/useScreenSize/breakpoints.js

```diff
@@ -0,0 +1,17 @@
+import xsBreakpoint from '@patternfly/react-tokens/dist/esm/global_breakpoint_xs';
+import smBreakpoint from '@patternfly/react-tokens/dist/esm/global_breakpoint_sm';
+import mdBreakpoint from '@patternfly/react-tokens/dist/esm/global_breakpoint_md';
+import lgBreakpoint from '@patternfly/react-tokens/dist/esm/global_breakpoint_lg';
+import xlBreakpoint from '@patternfly/react-tokens/dist/esm/global_breakpoint_xl';
+import xxlBreakpoint from '@patternfly/react-tokens/dist/esm/global_breakpoint_2xl';
```

Comment:
You want to import from the js version of the build. Otherwise, you might break nodejs env (tests) which will require transpiling the modules.